wiki/The_Team/es.md

```diff
@@ -52,14 +52,17 @@ Las personas que nombradas a continuación son el núcleo del equipo de osu!, y
 
 ## Grupos de Usuarios
 
+Los siguientes son grupos de usuarios compuestos por miembros de la comunidad que ayudan a mantener osu!
+La mayoría de estos grupos de usuarios reciben distintivos colores en el nombre dentro del foro, colores de chat en el juego y/o de perfil.
+
 | Nombre | Descripción |
 | ---- | ----------- |
-| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | Vigilando los foros y el chat en el juego |
-| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | Encargados de la calificación de beatmaps |
+| [Global Moderation Team](/wiki/The_Team/Global_Moderation_Team) | Vigilando los foros y el chat en el juego |
+| [Beatmap Nomination Group](/wiki/The_Team/Beatmap_Nomination_Group) | Encargados de la calificación de beatmaps |
 | [Developers](https://osu.ppy.sh/groups/11) | Haciendo que el juego sea impresionante al agregar nuevas funciones y corregir los errores |
-| [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
-| [Support Team Redux](/wiki/People/Support_Team) | Asistencia en foros de ayuda |
-| [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | Te ayudamos a disfrutar de osu!, empezando por asegurar la calidad de los beatmaps. |
+| [osu! Alumni](/wiki/The_Team/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
+| [Support Team Redux](/wiki/The_Team/Support_Team) | Asistencia en foros de ayuda |
+| [Quality Assurance Team](/wiki/The_Team/Quality_Assurance_Team) | Te ayudamos a disfrutar de osu!, empezando por asegurar la calidad de los beatmaps. |
```

Comment:
same deal as my comment about "Beatmap_Nomination_Group", this redirect doesn't (and shouldn't) exist so you need to fix the link